src/os_posix/os_time.c

```diff
@@ -39,9 +47,10 @@ __wt_epoch(WT_SESSION_IMPL *session, struct timespec *tsp)
 
 	WT_SYSCALL_RETRY(gettimeofday(&v, NULL), ret);
 	if (ret == 0) {
-		tsp->tv_sec = v.tv_sec;
-		tsp->tv_nsec = v.tv_usec * WT_THOUSAND;
-		__wt_time_check_monotonic(session, tsp);
+		tmp.tv_sec = v.tv_sec;
+		tmp.tv_nsec = v.tv_usec * WT_THOUSAND;
+		__wt_time_check_monotonic(session, &tmp);
+		*tsp->tv_sec = tmp;
```

Comment:
This doesn't look right -- I'd expect `*tsp = tmp;`

Do we build and test anywhere with `gettimeofday` but not `clock_gettime`?